### DISCUSSION TITLE
Possible Log4j Scan hit on latest version of 2.3

### DISCUSSION THREAD

**Discussion by Fenr1r-g** [Security-Onion-Solutions/securityonion#11296] [General] [answered]
Hello all,

I searched through the old posts for log4j fixes before this and believe this is a new find that our tenable instance hit on

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/54542665/876af015-d80d-47a0-88ef-dea8d28fc4c2)

I initially believed this to be a false positive, but after pulling down the jar file and running a "Jar -tf <file> | grep Jndi" I found the vulnerable lookup class in it.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/54542665/4f0517da-4a45-447c-9ca2-f1157fb4cda3)

Not sure what this file is being used for, but would appreciate confirmation from the team before re-rolling this finding as a false positive.

**Comment by dougburks** [comment]
What is the output of the following?
```
cat /etc/soversion
```

**Reply by Fenr1r-g** [reply]
2.3.260

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/54542665/5538083a-2ad3-4678-8dd1-63308a897ea8)

**Reply by dougburks** [reply]
Where exactly are you seeing this on your 2.3.260 installation?

**Reply by Fenr1r-g** [reply]
/source/logstash/logstash-core/src/test/resources/org/logstash/plugins/logstash-input-java_input_example-0.0.1.jar

**Reply by dougburks** [reply]
There shouldn't be a `/source` directory on a default Security Onion 2.3.260 installation:
```
ls /source
ls: cannot access /source: No such file or directory
```

**Reply by Fenr1r-g** [reply]
This is an Azure VM. Sorry, I should've specified this ahead of time.

**Reply by dougburks** [reply, marked as answer]
In that case, yes, cloud images do have a `/source` directory. This is a snapshot of the source code from all major third-party products included with Security Onion. However, no services actually run from there. You can safely remove that folder.

**Reply by Fenr1r-g** [reply]
Awesome, thanks Doug!
